cms/templates/emails/utilities_bulk_update_done_message.txt

```diff
@@ -0,0 +1,14 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hello, {username}").format(username=username)}
+
+% if success:
+${_("Your bulk update on the {course_name} course has successfully completed. Previously published problems have been automatically republished.").format(course_name=course_name)}
```

Comment:
Check indents for conditional.  Make sure text itself is not then indented.  Check for syntax to fix or forget it and revert.

Reply:
Does make text indented, quickly looked for solution, and just reverted change. Added item to follow-up work checklist in Trello card